docs/self-hosted/kubernetes-maintenance.md

```diff
@@ -1,6 +1,9 @@
 # Maintenance
 ## Private Packagist Self-Hosted (Kubernetes)
 
+This page describes common maintenance tasks for Private Packagist Self-Hosted in a Kubernetes cluster installed with [kURL](kubernetes-embedded). 
+This page is not relevant if you installed the application with [Helm](kubernetes-helm). 
+
```

Comment:
Where would be put Helm maintenance tasks like describing steps about how to update the Helm chart?

Reply:
Sounds like a plan, just need to make sure that the links from the changelog still work

Reply:
Check, the `#updates` anchor is still in place, but I'll update it to point directly to the KOTS instructions when updating composer.json reference.